src/track/track.js

```diff
@@ -0,0 +1,33 @@
+//@flow
+export default class Track {
+  _id: string;
+  _active: boolean;
+  _type: string;
+  _language: string;
+
+  get id(): string {
+    return this._id;
+  }
+
+  get active(): boolean {
+    return this._active;
+  }
+
+  set active(value: boolean) {
+    this._active = value;
+  }
+
+  get type(): string {
+    return this._type;
+  }
+
+  get language(): string {
+    return this._language;
+  }
+
+  constructor(id: string, active: boolean, language: string) {
```

Comment:
About the _language_ property, isn't it something that common only to audio and text tracks? if so, it should be in the derived AudioTrack and TextTrack classes and not here.